qa/tasks/cephfs/fuse_mount.py

```diff
@@ -143,9 +144,13 @@ def _list_fuse_conns(self):
 
         self.client_remote.run(args=['sudo', 'modprobe', 'fuse'],
                                check_status=False)
-        self.client_remote.run(
-            args=["sudo", "mount", "-t", "fusectl", conn_dir, conn_dir],
-            check_status=False, timeout=(30))
+
+        mounts = self.client_remote.run(
```

Comment:
This looks more than just a _cleanup_. Also, could this result in a hang if the client is blocklisted?

Reply:
True. It'll better to read `/proc/self/mounts` in this case too. I am trying to get this code phrase added as a convenience method - https://github.com/ceph/teuthology/pull/1786. Waiting for review/merge of this PR now.

Reply:
ack

Reply:
@rishabh-d-dave ping for this and https://github.com/ceph/teuthology/pull/1786